schemas/0.0.1/profile.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ld4p.github.io/sinopia/schemas/0.0.1/profile.json",
+  "type": "object",
+  "title": "LC Profile Schema (by Sinopia)",
+  "description": "Profile, or array of Resource Templates with some top-level metadata.",
+  "version": "0.0.1",
+  "required": [ "Profile" ],
+  "properties": {
+    "Profile": {
+      "type": "object",
+      "title": "Profile" ,
+      "description": "Profile key always at top level of a LC BFE / Sinopia BFF Profile.",
+      "required": [
+        "date",
+        "description",
+        "id",
+        "resourceTemplates",
+        "title"
+      ],
+      "properties": {
```

Comment:
Are we no longer using `adherence` and `source` properties for Profiles metadata?

Reply:
these schemas reflect the profiles from LC verso, not the ones we aspire to.  There will be future versions with the sinopia specific stuff.

Reply:
Cool, I'll resolve this and merge your changes.